### PR TITLE
Add landing page for Base Bootcamp

### PR DESCRIPTION
**Pull request by amhayslipcb**
**What changed? Why?**
Added base.org/bootcamp landing page.
Added link to Bootcamp under Base Camp nav on docs site.

Screenshot 2023-09-21 at 1 40 23 PM
Screenshot 2023-09-21 at 1 41 16 PM

**Notes to reviewers**
Has been reviewed by marketing, design, legal and privacy

**How has it been tested?**
Manually

### CODE REVIEW

File: apps/web/src/components/Bootcamp/HowItWorks/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    number: '3',
+    title: 'Kickoff',
+    description: 'Gather virtually for our initial kick-off. We’ll pair you with your mentor (an experienced Smart Contract Developer) and you’ll start the program.',
+  },
+  {
+    number: '4',
+    title: 'Build!',
+    description: (
+      <div className="text-md font-sans text-white">
+        <p>Work through <a className="underline" target="_blank" href="https://docs.base.org/base-camp/docs/welcome">Base Camp</a>, meeting regularly with your mentor. You’ll build your final project during the final two weeks - a real-world dapp that you’ll demo virtually on Demo Day.
```
Comment (zencephalon) [inline apps/web/src/components/Bootcamp/HowItWorks/index.tsx:28]: Sorry, this ought to have a linter on it, I'll try to find and configure one. Anytime we use `target="blank"` for security reasons we also ought to include `rel="noreferrer noopener"`. Could U go through the links in this PR and make sure they all have that?

File: package-lock.json
Comment (zencephalon) [inline package-lock.json:1]: This should be deleted

File: apps/web/src/components/Bootcamp/HowItWorks/index.tsx (hunk 1 above)
Comment (amhayslipcb) [reply apps/web/src/components/Bootcamp/HowItWorks/index.tsx:28]: For sure - thanks!

Comment (zencephalon) [reply apps/web/src/components/Bootcamp/HowItWorks/index.tsx:28]: Thanks so much, again sorry for the inconvenience, this ought to be a linter autofix in the future

Comment (amhayslipcb) [reply apps/web/src/components/Bootcamp/HowItWorks/index.tsx:28]: Weird - it looks like I (something) added it to some of them?

File: apps/web/pages/bootcamp.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        <meta
+          content="Base is a secure, low-cost, builder-friendly Ethereum L2 built to bring the next billion users onchain."
+          name="description"
+        />
+        <meta property="og:site_name" content="Base" />
+        <meta property="og:type" content="website" />
+        <meta property="og:title" content="Base" />
+        <meta
+          property="og:description"
+          content="Base is a secure, low-cost, builder-friendly Ethereum L2 built to bring the next billion users onchain."
```
Comment (zencephalon) [inline apps/web/pages/bootcamp.tsx:24]: Do we want this to be somewhat more specific to the bootcamp page at all?

**Review by zencephalon — APPROVED** [review]
Just one question

**Review by zencephalon — APPROVED** [review]

**Pull request merged**